region/aggregate-regions.yaml

```
@@ -15,13 +15,29 @@ Europe (excl. Turkey):
 Europe (incl. Turkey):
    definition: 'EU27 + EFTA + AL + BA + BY + ME + MK + UA + UK + RS + TR + XK'
 
+Pan-Europe:
+   definition: 'EU27 (excl. Malta & Cyprus) +CH + BA + ME + RS + MK + AL + NO + UK'
+   source: definition of the pan-european electricity system used in e-highways2050 and plan4res
+   
 # add more defintion of `Europe (*)` above this line
 
 EFTA:
    definition: 'IS + LI + NO + CH'
    notes: 'European Free Trade Association'
    source: 'https://en.wikipedia.org/wiki/European_Free_Trade_Association'
 
+BALKAN:
+   definition: 'SI+RO+HR+BA+ME+RS+BG+MK+GR+AL'
```

suggested change to follow three ideas:
 - use "readable" names 
 - order individual countries alphabetically
 - add spaces around operators (this is Python convention because it improves readability)

if you want to maintain the names currently used (BALKAN, SCAND) for later reference (or use in automated conversions), you could add a "synonyms" attribute as is used in `countries.yaml`.

```suggestion
Balkans:
   definition: 'AL + BA + BG + GR + HR + ME + MK + RO + RS + SI'
```